doc/rbd/rbd-exclusive-locks.rst

```diff
@@ -0,0 +1,78 @@
+.. _rbd-exclusive-locks:
+
+====================
+ RBD Exclusive Locks
+====================
+
+.. index:: Ceph Block Device; RBD exclusive locks; exclusive-lock
+
+Exclusive locks are a mechanism designed to prevent multiple processes
+from accessing the same Rados Block Device (RBD) in an uncoordinated
+fashion. Exclusive locks are heavily used in virtualization (where
+they enable live migration, among other things), and also in RBD
+mirroring (where they are a prerequisite for journaling).
+
+Exclusive locks are only available in RBD format version 2. They are
```

Comment:
"RBD image format version 2" -- but we also don't support creation of v1 images for several releases now.

Reply:
Yeah, I was wondering whether I should even mention it. Guess I'll just drop this reference, too.